api/DOMPoint.json

```diff
@@ -89,6 +89,51 @@
             "deprecated": false
           }
         }
+      },
+      "worker_support": {
+        "__compat": {
+          "description": "Available in workers",
+          "support": {
+            "chrome": {
+              "version_added": "58"
+            },
+            "chrome_android": {
+              "version_added": "58"
+            },
+            "edge": {
+              "version_added": false
+            },
+            "firefox": {
+              "version_added": "69"
+            },
+            "firefox_android": {
+              "version_added": false
+            },
+            "ie": {
+              "version_added": false
+            },
+            "opera": {
+              "version_added": true
+            },
+            "opera_android": {
+              "version_added": true
+            },
+            "safari": {
+              "version_added": "10.1"
+            },
+            "safari_ios": {
+              "version_added": "10.1"
```

Comment:
```suggestion
              "version_added": "10.3"
```

Reply:
1. Okay. I don't know where you find that information, so I didn't fill it out. Can you tell me where to look it up? I'd like to add it to the guide to finding version information on the MDN contributor guide.

2. Fixed in the update coming in a few minutes.

3. `DOMQuad` actually landed in [changeset 218458](https://trac.webkit.org/changeset/218458/webkit) for [WebKit bug 163534](https://bugs.webkit.org/show_bug.cgi?id=163534). If you browse the [code for iOS 10.3](https://trac.webkit.org/browser/webkit/releases/Apple/iOS%2010.3.3/WebCore/dom), `DOMQuadl.idl` is not present; it doesn't appear [until iOS 11](https://trac.webkit.org/browser/webkit/releases/Apple/iOS%2011.0/WebCore/dom). **So 11 is correct.**

_**PS:** This makes me think that we should have named `safari_ios` simply `ios`. Would have been clearer, I think. Plus would have helped with the ability to support other WebKit-based browsers on iOS._

Reply:
Regarding the Opera, Opera Android, and Samsung Internet versions, sorry for not specifying my data source!  I compared the engine versions in the files in our browsers/ folder.

Thanks for clarifying the iOS version by the way!  Regarding a rename, I’d say it’s definitely worth bringing up further discussion.  😉